### ISSUE TITLE
SA1115 should allow comments between parameters.

### ISSUE THREAD

**Issue by vweijsters** [DotNetAnalyzers/StyleCopAnalyzers#1921]
The following code should not produce SA1115, but it currently does:

```csharp
    public void MyTest(int v1, int v2, int v3)
    {
    }

    public void TestMethod()
    {
        this.MyTest(
            1,
            /* comment */
            2,
            /* comment */
            3);
    }
```

The same construction should probably also be allowed for `////` comments. `//` comments will not work because they require an empty line before them and that will (and shall) trigger SA1115.

**Comment by sharwell** [comment]
What does StyleCop Classic do in this case?

**Comment by vweijsters** [comment]
The code above is accepted by StyleCop Classic without warnings.

**Comment by sharwell** [comment]
Thanks, changing this to a :bug:.

**Comment by vweijsters** [comment]
Grabbing this
